schemas/fragments/TransformInput.json

```diff
@@ -16,7 +16,12 @@
     "name": {
       "type": "string",
       "format": "dataset-name",
-      "description": "An alias of this input to be used in queries."
+      "description": "A name of this input's table to be used in queries."
+    },
+    "dataset_ref": {
```

Comment:
All fields names in our JSON schemas are `snakeCase` (we should really add a schema linter to catch such things).

Reply:
Fixed